From 1b0a228b03ec5e3317cd587ae6ad475732cdb9aa Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Thu, 30 Oct 2008 14:31:23 +1100
Subject: [PATCH] netbase: provide a default network setup for palmt650 so usbnet works

palmt650 system didnt have a specirfic netbase setup to make usbnet work.
this provides that and makes it work.
---
 packages/netbase/netbase/palmt650/interfaces |   22 ++++++++++++++++++++++
 1 files changed, 22 insertions(+), 0 deletions(-)
 create mode 100644 packages/netbase/netbase/palmt650/interfaces

diff --git a/packages/netbase/netbase/palmt650/interfaces b/packages/netbase/netbase/palmt650/interfaces
new file mode 100644
index 0000000..db4c4ca
--- /dev/null
+++ b/packages/netbase/netbase/palmt650/interfaces
@@ -0,0 +1,22 @@
+# /etc/network/interfaces -- configuration file for ifup(8), ifdown(8)
+
+# The loopback interface
+auto lo
+iface lo inet loopback
+
+# Wireless interfaces
+
+# Wired or wireless interfaces
+
+# Ethernet/RNDIS gadget (g_ether)
+# ... or on host side, usbnet and random hwaddr
+auto usb0
+iface usb0 inet static
+	address 192.168.0.202
+	netmask 255.255.255.0
+	network 192.168.0.0
+	gateway 192.168.0.200
+
+# Bluetooth networking
+iface bnep0 inet dhcp
+
-- 
1.5.4.3

